src/ui/components/Tour/Tour.tsx

```diff
@@ -157,19 +157,31 @@ const Tour: React.FC = () => {
         )}
 
         {!isNewUser && showConsoleNavigate && showBreakpointAdd && showBreakpointEdit && (
-          <img src="/images/tour/fast-forward.gif" className={styles.videoExample} />
+          <img
+            src="https://website-git-master-recordreplay.vercel.app/tour/fast-forward.gif"
```

Comment:
These URLs still look wrong.

I don't know what "website-git-master-recordreplay.vercel.app" or how to deploy things there.

Reply:
Good call, we're going to standardise on https://vercel.replay.io/.

We're doing it this way to avoid having a ton of big media files in devtools. These are pushed via the website repo.